osmosis-proto-build/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "osmosis-proto-build"
```

Comment:
I'd suggest just putting this in the existing `proto-build`, possibly isolated into its own module.

Copying/pasting the code creates more maintenance effort.

Reply:
sure, I will check if I can merge it with `proto-build` adding a `osmosis` feature tag.